projects/uitdatabank/models/place-with-write-example.json

```diff
@@ -0,0 +1,143 @@
+{
+  "title": "place",
+  "$ref": "./place.json",
+  "examples": [
+    {
+      "mainLanguage": "nl",
+      "name": {
+        "nl": "Nederlandse naam",
+        "fr": "Nom français",
+        "de": "Deutscher Name",
+        "en": "English name"
+      },
+      "address": {
+        "nl": {
+          "addressCountry": "BE",
+          "addressLocality": "Brussel",
+          "postalCode": "1000",
+          "streetAddress": "Wetstraat 1"
+        },
+        "fr": {
+          "addressCountry": "BE",
+          "addressLocality": "Bruxelles",
+          "postalCode": "1000",
+          "streetAddress": "Rue de la Loi 1"
+        },
+        "de": {
+          "addressCountry": "BE",
+          "addressLocality": "Brüssel",
+          "postalCode": "1000",
+          "streetAddress": "Wetstraat 1"
+        },
+        "en": {
+          "addressCountry": "BE",
+          "addressLocality": "Brussels",
+          "postalCode": "1000",
+          "streetAddress": "Wetstraat 1"
+        }
+      },
+      "status": {
+        "type": "Unavailable",
+        "reason": {
+          "nl": "Nederlandse reden",
+          "fr": "Raison français",
+          "de": "Deutscher Grund",
+          "en": "English reason"
+        }
+      },
+      "bookingAvailability": {
```

Comment:
This is not supported for places, so best to omit this from the example body.

Reply:
Thanks @corneelwille , you are correct. But I've taken these examples / descriptions from previously existing endpoints, and I've even indirectly referenced them to avoid duplicates of those models/examples. So I'd suggest that we merge this and then fix those descriptions / examples in another PR, to avoid this PR remaining open due to unrelated issues that are not caused by this PR in itself.